packages/block-library/src/post-template/edit.js

```diff
@@ -95,6 +95,11 @@ export default function PostTemplateEdit( {
 	const [ { page } ] = queryContext;
 	const [ activeBlockContextId, setActiveBlockContextId ] = useState();
 
+	const { records: categories } = useEntityRecords( 'taxonomy', 'category', {
```

Comment:
I guess we could fetch only the specific term instead of all, no?

Reply:
I expected this to work by not throwing a network request if the data was available locally but, apparently, it does anyway. So, in addition to only request the target category, I've limited the query by fields, so we only get the data we need.